samples/chat.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from absl.testing import absltest
+
+import google.generativeai as genai
+import pathlib
+
+media = pathlib.Path(__file__).parents[1] / "third_party"
+
+class UnitTests(absltest.TestCase):
+    def test_chat(self):
+        # [START chat]
+        model = genai.GenerativeModel("gemini-1.5-flash")
+        chat = model.start_chat(
```

Comment:
It's good that we're showing how to set the history, but I think at minimum one of the examples should show more than one round of chat.